app/views/reject_case_reason.scala.html

```diff
@@ -70,10 +71,7 @@
             customErrorMessage = Some(messages("error.empty.reject.note"))
         )
 
-        @warning(
-            "This case cannot be reopened after it has been rejected.",
```

Comment:
Moved this and all other messages that were updated to the messages file. Although this is a stride FE and we don't need welsh, the messages file should be used instead of hardcoding the english text into the html as it was done here and other places.